package.json

```diff
@@ -73,6 +73,7 @@
     "aria-live-storybook-addon": "^0.0.6",
     "autoprefixer": "^10.4.7",
     "axe-core": "^4.3.3",
+    "classnames": "^2.3.2",
```

Comment:
I added this dependency now, it isn't being used, but it will be used soon, so... I'll keep it here. :)